code/game/objects/items/holy_weapons.dm

```diff
@@ -386,31 +386,35 @@
 	icon = 'icons/obj/transforming_energy.dmi'
 	icon_state = "swordblue"
 	inhand_icon_state = "swordblue"
-	desc = "If you strike me down, I shall become more robust than you can possibly imagine."
+	worn_icon_state = "swordblue"
 	slot_flags = ITEM_SLOT_BELT
+	desc = "If you strike me down, I shall become more robust than you can possibly imagine."
 
 /obj/item/nullrod/claymore/saber/red
 	name = "dark energy sword"
 	icon_state = "swordred"
 	inhand_icon_state = "swordred"
+	worn_icon_state = "swordred"
 	desc = "Woefully ineffective when used on steep terrain."
```

Comment:
while we're in the area, I'm kind of wondering if "on" (instead of "in") is the right word to use here

also, it'd be absolutely hilarious if we added some sort of multi-z mechanic effect to this description, but again, that'd be for another PR.